src/components/fields/AxisCreator.js

```diff
@@ -0,0 +1,155 @@
+import Dropdown from './Dropdown';
+import Field from './Field';
+import Info from './Info';
+import PropTypes from 'prop-types';
+import React, {Component, Fragment} from 'react';
+import {connectToContainer, localize, traceTypeToAxisType} from 'lib';
+import {EDITOR_ACTIONS} from 'lib/constants';
+import {PlusIcon} from 'plotly-icons';
+
```

Comment:
this component should show the axis title, not just the `x`, `x2`, `y2` axis IDs

Reply:
that's done, as per below gif

Reply:
It's done in the `AxesSelector` but not the `AxisCreator` right?

Reply:
right..